roles/serviceassurance/tasks/main.yml

```diff
@@ -1,8 +1,8 @@
 ---
 - name: Execute default tasks when metrics are requested to be enabled
   include_tasks: sa_metrics.yml
-  when: metrics.enabled
+  when: metrics_enabled
```

Comment:
If you're wondering why the mix of snake_case and camelCase:

> Defining the spec for an Ansible-based Operator can be done entirely in Ansible. The Ansible Operator passes all 
> key-value pairs listed in the CR spec field along to Ansible as variables. The names of all variables in the spec 
> field are converted to snake case (lowercase with an underscore) by the Operator before running Ansible. For 
> example, serviceAccount in the spec becomes service_account in Ansible.

[source: https://docs.openshift.com/container-platform/4.1/applications/operator_sdk/osdk-ansible.html#osdk-building-ansible-operator_osdk-ansible] bullet 3